lib/internal/encoding.js

```diff
@@ -327,7 +327,7 @@ class TextEncoder {
     });
     obj.encoding = this.encoding;
     // Lazy to avoid circular dependency
-    return require('util').inspect(obj, opts);
+    return require('internal/util/inspect').inspect(obj, opts);
```

Comment:
If it was possible to change
https://github.com/nodejs/node/blob/1329d0a1582fe35628a914aed699827d3415f3fc/lib/internal/encoding.js#L21
to
```js
customInspectSymbol: <something else>
```
I could put `require('internal/util/inspect').inspect` together with all the top level imports.